hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/tasks/OMDBUpdatesHandler.java

```diff
@@ -93,6 +93,18 @@ private void processEvent(int cfIndex, byte[] keyBytes, byte[]
       valueBytes, OMDBUpdateEvent.OMDBUpdateAction action)
       throws IOException {
     String tableName = tablesNames.get(cfIndex);
+    // DTOKEN_TABLE is using OzoneTokenIdentifier as key instead of String
+    // and assuming to typecast as String while de-serializing will throw error.
+    // omdbLatestUpdateEvents defines map key as String type to store in its map
+    // and to change to Object as key will have larger impact considering all
+    // ReconOmTasks. Currently, this table is not needed to sync in Recon OM DB
+    // snapshot as this table data not being used currently in Recon.
+    // When this table data will be needed, all events for this table will be
+    // saved using Object as key and new task will also retrieve using Object
+    // as key.
+    if (OMDBDefinition.DTOKEN_TABLE.getName().equalsIgnoreCase(tableName)) {
+      return;
+    }
```

Comment:
Hi @devmadhuu ,  can we something like this? 

Object key = codecRegistry.asObject(keyBytes, keyType.get());
builder.setKey(key instanceof String? key : key.toString());

I'm afraid that we filter DTOKEN_TABLE this time. Next time, if there is a new table added in OM which doesn't use String as key, then this filter logic need to be updated again.

Reply:
@ChenSammi  - I have done changes as suggested. Pls re-review.